### DISCUSSION TITLE
Endless xlsx file download

### DISCUSSION THREAD

**Discussion by Solomon1228** [eclipse-birt/birt#1878] [Q&A] [answered]
Good afternoon, sorry for the bad English.
faced with such a problem. When using view report as xlsx, the report is loaded endlessly on the report, while the other functions work properly.
Birt version 4.15
Java version 21.0.3
Eclipse version 2024 06 R

**Comment by merks** [comment, marked as answer]
If there is a bug, we need to fix in in 4.17 and we'd need a test case that reproduces the problem.  I did recently fix a similar sounding problem in 4.17

https://download.eclipse.org/birt/updates/nightly/latest/

The 4.17 version will be released very soon in September so now is the perfect time to help test it and report issues that are reproducible with that version to ensure that the 4.17 release works well for you.

**Comment by Solomon1228** [comment]
Wow, I tried 4.17, and many problems are gone, now I will test and write if I find something else.

Thanks a lot

**Reply by merks** [reply]
That's great to hear!  Thanks for testing!  ❤️
